README.md

```diff
@@ -103,8 +103,8 @@ Please note for this to work you should create/update `~/.gradle/gradle.properti
     signing.password=
     signing.secretKeyRingFile=
 
-### Publishing the jars without signing to a local repository ###
-    ./gradlew -Dorg.gradle.project.skipSigning=true -Dorg.gradle.project.mavenUrl=file://path/to/repo uploadArchivesAll
+### Installing the jars to the local Maven repository ###
```

Comment:
Yeah, I should use `installAll` instead, good point. The reason for removing this (as discussed offline) is that it's easy to use it instead of `installAll` by mistake and explaining the difference seemed like it was not worth it.